src/kili/services/asset_import/base.py

```diff
@@ -441,6 +442,13 @@ def check_mime_type_compatibility(self, path: str):
         if not os.path.isfile(path):
             raise FileNotFoundError(f"file {path} does not exist")
         mime_type, _ = mimetypes.guess_type(path)
+        if mime_type is None and platform.system() == "Windows":
+            try:
+                import magic
+
+                mime_type = magic.from_file(path, mime=True)
```

Comment:
Not sure if `magic.from_file` returns something else than `None` or an error in case it does not find the mime type but I was not able to find the answer neither in their doc (quite poor) or in the code :(

Reply:
yep not sure how we can do this easily...

https://github.com/ahupp/python-magic/pull/294 could help